<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveCaptureWidget</class>
 <widget class="QWidget" name="LiveCaptureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="cbCameras"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnStartLive">
         <property name="text">
          <string>Start Livestream</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStopLive">
         <property name="text">
          <string>Stop Livestream</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnSaveDir">
         <property name="text">
          <string>Save Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSaveDir">
         <property name="text">
          <string>/tmp</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnStartRec">
         <property name="text">
          <string>Start Recording</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStopRec">
         <property name="text">
          <string>Stop Recording</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="btnStartExtr">
         <property name="text">
          <string>Start Extracting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStopExtr">
         <property name="text">
          <string>Stop Extracting</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnLoadAreas">
         <property name="text">
          <string>Load Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblLoadAreas">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="infoText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
